Makefile.include

```diff
@@ -302,3 +302,6 @@ else # RIOT_VERSION
 	$(MAKE) RIOTBASE=$(<D) $(filter-out clean, ${MAKECMDGOALS})
 
 endif
+
+help:
+	@$(MAKE) -qp | sed -ne 's/\(^[a-z][a-z_-]*\):.*/\1/p' | sort | uniq
```

Comment:
is the `.*` always greedy? 
otherwise: should we add a `$` to the end of the match pattern?